src/GafferUI/Pointer.cpp

```diff
@@ -76,6 +76,9 @@ static Registry &registry()
 		r["pivot"] = new Pointer( "pointerPivot.png", Imath::V2i( 13, 0 ) );
 		r["cut"] = new Pointer( "pointerCut.png", Imath::V2i( 11, 7 ) );
 		r["notEditable"] = new Pointer( "pointerNotEditable.png", Imath::V2i( 10 ) );
+		r["replacePaths"] = new Pointer( "pointerReplacePaths.png", Imath::V2i( 21, 6 ) );
+		r["addPaths"] = new Pointer( "pointerAddPaths.png", Imath::V2i( 21, 6 ) );
+		r["removePaths"] = new Pointer( "pointerRemovePaths.png", Imath::V2i( 21, 6 ) );
```

Comment:
When we have more specific icons, it would be more appropriate to do this registration in GafferSceneUI - these icons are all expected to be generic enough to make sense in GafferUI alone.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5786/commits/a43f45d1b33882bca64fb96f0e8c61cd1b0c300b